tests/cue/readme.md

```diff
@@ -15,4 +15,6 @@ Each entry of the document is structured as follows:
     * `re`: custom regular expression to extract the version number from the command output, default is `r'(?:(\d+\.(?:\d+\.)*\d+))'`
     * `modname`: load the specified module before executing the test. Packages requiring a module load in certain sites and are installed at system level in others can be handled by greedly adding the required module in this field. 
 
-## env.py
\ No newline at end of file
+## env.py
+
+## shared_fs.py
```

Comment:
Description in the readme ?

Reply:
https://github.com/vscentrum/vsc-test-suite/pull/8/commits/9f50bfdc719cbb0028b6ad97039018b23095f3e2 solves the issues